### PR TITLE
Implanter Fabricator

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR
Adds a new Lathe that allows for the production of various implants. This automaton can be Emagged to gain access to some not-so-strong implants.

Removed the ability to buy fun implants via cargo.

## Why / Balance
Implants are a rather interesting mechanic that is very underutilized

I think we also need useful implants for the crews. Adding this to the medical machine would increase the variety of gameplay of the medical department.

## Technical details
no c#

## Media
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added new ImpLathe. Lathe to fabricating implanters.
- remove: Cargo can't buy implants anymore.

### CODE REVIEW

**Comment by TheShuEd** [comment]
I wanted to make biochem cooler, but you nuke it

**Comment by metalgearsloth** [comment]
I think inventory is interesting and nuking it for powergaming implants doesn't seem good and just creeps them even harder. The only reason nukies / sec have implants is important to bypass inventory but there's no reason a random clown needs to be able to teleport at will.

**Comment by deltanedas** [comment]
> there's no reason a random clown needs to be able to teleport at will.

thats emag only

**Comment by metalgearsloth** [comment]
> > there's no reason a random clown needs to be able to teleport at will.
> 
> thats emag only

It's still forcing implant removal as a mechanic and bypassing the entire reason it even exists for nukies.

**Comment by Aisu9** [comment]
why emag card need to gain more value again?

**Comment by Kadeo64** [comment]
I don't think PKA implant being random is a good idea, it should probably just have a long cooldown (15 seconds maybe)

File: Resources/Prototypes/Recipes/Lathes/implants.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  result: ProtokineticImplanter
+  completetime: 5
+  materials:
+    Plastic: 100
+    Glass: 150
+    Silver: 100
+    Plasma: 150
+
+# Antag
+
```
Comment (keronshb) [inline Resources/Prototypes/Recipes/Lathes/implants.yml:81]: These shouldn't be printable, even by emag. It's huge powercreep and they're meant to be bought via uplink.

File: Resources/Prototypes/Recipes/Lathes/implants.yml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  icon:
+      sprite: Objects/Specific/Medical/implanter.rsi
+      state: implanter0
+  result: MindShieldImplanter
+  completetime: 5
+  materials:
+    Plastic: 100
+    Glass: 150
+    Silver: 100
+    Plasma: 50
```
Comment (keronshb) [inline Resources/Prototypes/Recipes/Lathes/implants.yml:65]: Mindshields should not be printable.

**Review by keronshb — CHANGES_REQUESTED** [review]

**Comment by Hmeister-real** [comment]
o7

**Pull request closed without merging**